services/todo-legacy/datasources.ts

```diff
@@ -0,0 +1,12 @@
+export const datasources: Object = {
```

Comment:
Do we want a type definition for a one-off object? If we intend to make a DataSourceOptions class/interface in `@loopback/*`, then that makes sense.